packages/MarkdownEditor-Tests.package/MarkdownInlineTextStylerTest.class/instance/testInterpretTokensAsItalics.st

```diff
@@ -1,11 +1,11 @@
 tests - interpretation
 testInterpretTokensAsItalics
 
+	| expectedEmphases |
```

Comment:
I don't see the advantage of an inline variable instead of inlining the array and putting some breaks in the assert call.